custom_components/tuya_local/devices/xoca_dac2121c_energy_meter.yaml

```diff
@@ -13,9 +13,14 @@ primary_entity:
       type: base64
       optional: true
       force: true
+    - id: 6
```

Comment:
The id changed from `17` to `6` in the move up to here. Was it intentional?

Reply:
ooops. usual copy/paste issue :)